packages/oui-switch/src/switch.controller.js

```diff
@@ -0,0 +1,18 @@
+import { addBooleanParameter } from '@oui-angular/common/component-utils'
+
+export default class {
+  constructor ($scope, $attrs, $transclude) {
+    'ngInject'
+
+    this.$scope = $scope
+    this.$attrs = $attrs
+  }
+
+  $onInit () {
+    addBooleanParameter(this, 'disabled')
+
+    if (!this.name) {
```

Comment:
maybe addStringParameter with a default value? (like addBooleanParameter)

Reply:
Could be nicer indeed!